proxy/client.go

```diff
@@ -0,0 +1,226 @@
+package proxy
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"math/big"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/rollkit/go-execution"
+	proxy_json_rpc "github.com/rollkit/go-execution/proxy/jsonrpc"
+	rollkitTypes "github.com/rollkit/rollkit/types"
```

Comment:
Import aliases should be in lower case (or at least should have same format everywhere).